.travis.yml

```diff
@@ -1,14 +0,0 @@
-language: python
```

Comment:
I wonder why the travis checks are still showing on this PR.

Reply:
I think because they're still in master?